### PR TITLE
Address 219

### PR DESCRIPTION
**Pull request by nearmax**

### CODE REVIEW

File: environment/lib/near2eth-relay.js [hunk 1]
```diff
@@ -237,22 +237,10 @@ class Near2EthRelay {
                 }
                 try {
                     lightClientBlock = await near.connection.provider.sendJsonRpc('next_light_client_block', [clientBlockHash]);
-                    break;
-                } catch (err) {
```
Comment (ailisp) [inline environment/lib/near2eth-relay.js:241]: i suggest keep this catch, so we can tell it's near sendJsonRpc error or web3 error:
```
for ... {
    try {
            lightClientBlock = await near.connection.provider.sendJsonRpc('next_light_client_block', [clientBlockHash]);
    catch(err) {...}
    try{
        await clientContract.methods.addLightClientBlock(borshBlock)...
        break
    }catch(err) {...}
}
```

Comment (nearmax) [reply environment/lib/near2eth-relay.js:241]: The problem with this catch is that if for whatever reason we are getting wrong header and client rejects it we will not re-fetch the new header and will be trying to resubmit the broken header over and over.

Comment (ailisp) [reply environment/lib/near2eth-relay.js:241]: In the second catch put a `continue`?

**Comment by nearmax** [comment]
I don't think it is needed anymore. Closing.

**Pull request closed without merging**